Libraries/src/Amazon.Lambda.LexV2Events/LexV2ActiveContext.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System.Collections.Generic;
+using System.Runtime.Serialization;
+
+namespace Amazon.Lambda.LexV2Events
+{
+    /// <summary>
+    /// Contains information about the contexts that a user is using in a session.
+    /// https://docs.aws.amazon.com/lexv2/latest/dg/API_runtime_ActiveContext.html
+    /// </summary>
+    [DataContract]
+    public class LexV2ActiveContext
```

Comment:
Looking more about the pattern, we have been inconsistent in past too

For example `[SlotDetail](https://github.com/aws/aws-lambda-dotnet/blob/c1ce1e31d5f3d256f1829586f476be51b510e313/Libraries/src/Amazon.Lambda.LexEvents/LexEvent.cs#L152)`, it not called `LexSlotDetail` 

However `[LexCurrentIntent](https://github.com/aws/aws-lambda-dotnet/blob/c1ce1e31d5f3d256f1829586f476be51b510e313/Libraries/src/Amazon.Lambda.LexEvents/LexEvent.cs#L118)` has prefix Lex.

Reply:
Not sure why `SlotDetail` didn't have prefix but the predominated pattern used across all of the event packages is to prefix with the service name. Many of the event packages use inner classes and the inner classes don't use prefixes. I think we should keep the `LexV2` since there is more precedent for having prefixes.

Reply:
@ganeshnj @normj Please confirm if no change is required for this review comment as we intend to keep LexV2 prefix.

Reply:
Keep the `LexV2` prefix, because we have done it in past.